.ci/install_firecracker.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$(arch)" != "x86_64" ]; then
+	die "Static binaries for Firecracker only available with x86_64."
+	exit 1
+fi
+
+docker_version=$(docker version --format '{{.Server.Version}}' | cut -d '.' -f1-2)
+
+if [ "$docker_version" != "18.06" ]; then
+	die "Firecracker hypervisor only works with docker 18.06"
+	exit 1
+fi
+
+release_version="1.5.0-rc2"
+url="https://github.com/kata-containers/runtime/releases/download/$release_version/kata-fc-static-$release_version-$arch.tar.gz"
+echo "Get static binaries"
+curl -OkL ${url}
+
+echo "Decompress binaries"
+sudo tar -xvf kata-fc-static-$release_version-$arch.tar.gz -C /
+
+echo "Install and configure docker"
+docker_configuration_path="/etc/docker"
+# Check if directory exists
+if [ ! -d $docker_configuration_path ]; then
+	sudo mkdir $docker_configuration_path
+fi
+
+# Check if daemon.json exists
+docker_configuration_file=$docker_configuration_path/daemon.json
+
+if [ -f $docker_configuration_file ]; then
+	# Check devicemapper flag
+	check_devicemapper=$(grep -w '"storage-driver": "devicemapper"' $docker_configuration_file | wc -l)
+	if [ $check_devicemapper -eq 0 ]; then
+		die "Devicemapper is not enabled at $docker_configuration_file"
+		exit 1
+	fi
+	# Check kata runtime flag
+	check_kata=$(grep -w '"path": "/opt/kata/bin/kata-runtime"' $docker_configuration_file | wc -l)
+	if [ $check_kata -eq 0 ]; then
+		die "Kata Runtime path not found at $docker_configuration_file"
+		exit 1
+	fi
+else
+	sudo touch $docker_configuration_file
+	sudo bash -c 'cat > /etc/docker/daemon.json <<-EOF
```

Comment:
Shouldn't this be referencing `$docker_configuration_file` ?

Also, it can be simplified as:

```sh
cat <<-EOF | sudo tee "$docker_configuration_file"
   ...
EOT
```